core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -827,6 +828,32 @@
           + " need fast seeks within the timestamp range of a column. When setting this to fail, "
           + "also consider configuring the `" + NoDeleteConstraint.class.getName() + "` "
           + "constraint."),
+  @Experimental
+  TABLE_HDFS_POLICY_PREFIX("table.hdfs.policy.", null, PropertyType.PREFIX,
```

Comment:
Do you expect the user to be able to set arbitrary properties of the form `table.hdfs.policy.X` where `X` is an arbitrary user string?

Reply:
Yeah, I was a little conflicted about this one while fixing up the unit tests.  I like having the prefix so it's a little easier to check if properties are related to policy, but since there are only 2 policies I could just enumerate them.

Reply:
I was curious if removing this prefix would simplify or remove some of the validation code added.  With two enumerated properties, its possible no additional validation code will be needed because the existing validation for specific properties can be used.

Reply:
I'll remove it since it doesn't really fit with the other prefixes.  that will definitely simplify the unit tests, at least